### DISCUSSION TITLE
Midi Sysex from Buttons

### DISCUSSION THREAD

**Discussion by JimMcDougall** [shanteacontrols/OpenDeck#305] [Ideas]
Hi Paradajz,
I would like to be able to send Midi Sysex messages from Buttons. examples - Bank select as I have multiple synths with multiple banks, Also to switch between Voice and Combinations for MPE. 
Separate question, you show supporting the SSD1306 but I have been led to believe the SSD1309 can be a direct substitute. Will the ssd1309 work in OpenDeck?
Thank you

**Comment by paradajz** [comment]
> Hi Paradajz, I would like to be able to send Midi Sysex messages from Buttons. examples - Bank select as I have multiple synths with multiple banks, Also to switch between Voice and Combinations for MPE.

That's not supported.

> Separate question, you show supporting the SSD1306 but I have been led to believe the SSD1309 can be a direct substitute. Will the ssd1309 work in OpenDeck? Thank you

It can work yes but it's not supported and I have no way of testing it since I don't have that display.
